docs/sources/setup-grafana/configure-security/configure-authentication/azuread/index.md

```diff
@@ -66,55 +71,85 @@ To enable the Azure AD OAuth2, register your application with Azure AD.
 
 1. Click **Add** then copy the key value. This is the OAuth client secret.
 
-1. Click **Manifest**, then define the required Application Role values for Grafana: Viewer, Editor, or Admin. If not defined, all users will have the Viewer role. Every role requires a unique ID which you can generate on Linux with `uuidgen`, and on Windows through Microsoft PowerShell with `New-Guid`.
-
-1. Include the unique ID in the configuration file:
-
-   ```json
-   	"appRoles": [
-   			{
-   				"allowedMemberTypes": [
-   					"User"
-   				],
-   				"description": "Grafana org admin Users",
-   				"displayName": "Grafana Org Admin",
-   				"id": "SOME_UNIQUE_ID",
-   				"isEnabled": true,
-   				"lang": null,
-   				"origin": "Application",
-   				"value": "Admin"
-   			},
-   			{
-   				"allowedMemberTypes": [
-   					"User"
-   				],
-   				"description": "Grafana read only Users",
-   				"displayName": "Grafana Viewer",
-   				"id": "SOME_UNIQUE_ID",
-   				"isEnabled": true,
-   				"lang": null,
-   				"origin": "Application",
-   				"value": "Viewer"
-   			},
-   			{
-   				"allowedMemberTypes": [
-   					"User"
-   				],
-   				"description": "Grafana Editor Users",
-   				"displayName": "Grafana Editor",
-   				"id": "SOME_UNIQUE_ID",
-   				"isEnabled": true,
-   				"lang": null,
-   				"origin": "Application",
-   				"value": "Editor"
-   			}
-   		],
-   ```
-
-1. Go to **Azure Active Directory** and then to **Enterprise Applications**. Search for your application and click on it.
-
-1. Click on **Users and Groups** and add Users/Groups to the Grafana roles by using **Add User**.
-
+2. You can define the required Application Roles for Grafana [using the Azure Portal](#configure-application-roles-for-grafana-on-the-azure-portal) or [using the manifest file](#configure-application-roles-for-grafana-in-the-manifest-file).
+
+3. Go to **Azure Active Directory** and then to **Enterprise Applications**. Search for your application and click on it.
+
+4. Click on **Users and Groups** and add Users/Groups to the Grafana roles by using **Add User**.
```

Comment:
```suggestion
1. Click **Users and Groups**.

1. Click **Add User** to add a user or group to the Grafana roles.
```

Reply:
Replaced, but changed "Add user" to "Add user/group" to match the label of the button.